src/Microsoft.Framework.Configuration.Json/JsonConfigurationSource.cs

```diff
@@ -78,7 +80,20 @@ public override void Load()
         internal void Load(Stream stream)
         {
             JsonConfigurationFileParser parser = new JsonConfigurationFileParser();
-            Data = parser.Parse(stream);
+            try
+            {
+                Data = parser.Parse(stream);
+            }
+            catch(Exception e)
+            {
+                var splitErrorMessageArray = e.Message.Split(
+                    new string[] { _exceptionMessageSplitString },
```

Comment:
I cannot use `.Path` since its value will be `'address'` for an error message like this - `Unexpected end of content while loading JObject. Path 'address', line 7, position 44.` So if I split based on `Path` value, then error message will be - `Unexpected end of content while loading JObject. Path `